src/main/java/com/github/javafaker/BrazilianDocuments.java

```diff
@@ -0,0 +1,216 @@
+package com.github.javafaker;
```

Comment:
I'd prefer it if you put this class in its own package... maybe `com.github.javafaker.brazil`